<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HxD Cheap Clone</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="loadFileButton">
        <property name="text">
         <string>Load File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="openedFileLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="saveFileButton">
        <property name="text">
         <string>Save Changes</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="ContinuousEditionTableView" name="hexTable"/>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ContinuousEditionTableView</class>
   <extends>QTableView</extends>
   <header>continuouseditiontableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
